### PR TITLE
add security context to placeholder pod

### PR DESCRIPTION
**Pull request by OliverMKing**
# Description

adds security context to placeholder pod

### CODE REVIEW

File: pkg/controller/keyvault/placeholder_pod.go [hunk 1]
```diff
@@ -259,6 +260,20 @@ func (p *PlaceholderPodController) buildDeployment(ctx context.Context, dep *app
 							corev1.ResourceMemory: resource.MustParse("24Mi"),
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
+						Privileged:               util.ToPtr(false),
+						AllowPrivilegeEscalation: util.ToPtr(false),
+						Capabilities: &corev1.Capabilities{
+							Drop: []corev1.Capability{"ALL"},
```
Comment (jaiveerk) [inline pkg/controller/keyvault/placeholder_pod.go:267]: nice addition here

**Review by jaiveerk — APPROVED** [review]

**Review by bfoley13 — APPROVED** [review]
Looks good

**Review by davidgamero — APPROVED** [review]

**Comment by davidgamero** [comment]
lgtm

**Pull request merged**